data/json/npcs/talk_tags.json

```diff
@@ -1320,7 +1744,10 @@
       "Somebody get some water!",
       "Fire, fire, FIRE!",
       "Get an extinguisher!",
-      "Danger hot!"
+      "Danger hot!",
+      "The place is burning!",
+      "Things are gonna get real hot if we don't put that fire out right stat now!",
```

Comment:
either `that fire out, stat!` or `that fire out right <swear> now!`

Reply:
This one was a bit of a reference to this clip. https://www.youtube.com/watch?v=zS4InRg9rE4